landlab/plot/imshowhs.py

```diff
@@ -454,6 +456,8 @@ def _imshowhs_grid_values(
     cmap = plt.get_cmap(cmap)
 
     if color_for_closed is not None:
+        if color_for_closed != "white":
```

Comment:
Why limit *color_for_closed* to only *"white"* or *None*? It seems like an [matplotlib color](https://matplotlib.org/stable/tutorials/colors/colors.html#sphx-glr-tutorials-colors-colors-py) should work.

Reply:
Good point, to plot the hillshade with an extra layer, I first create an RGB array and next change the transparency before plotting using pyplot's imshow. Didn't know about a quick fix to make this working with masks. 
See: 
`rgb[:, :, 3][values.mask] = 0`
` ima = ax1.imshow(rgb, extent=extent, **kwds)`

Reply:
I think [to_rgba](https://matplotlib.org/stable/api/_as_gen/matplotlib.colors.to_rgba.html?highlight=to_rgba) should do what you want.
```python
>>> from matplotlib.colors import to_rgba
>>> to_rgba("w")
(1.0, 1.0, 1.0, 1)
>>> to_rgba("w", alpha=0.0)
(1.0, 1.0, 1.0, 0.0)
>>> to_rgba("c")
(0.0, 0.0, 0.0, 1)
>>> to_rgba("k")
(0.0, 0.75, 0.75, 1)
```

Reply:
Yup, that's it!